src/Sylius/Bundle/CoreBundle/Form/Type/Promotion/PromotionConfigurationType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type\Promotion;
+
+use Symfony\Component\Form\AbstractType;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+final class PromotionConfigurationType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
It should be ``getBlockPrefix`` - it also causes some errors, as there is no proper widget used to rendering action/rule configuration.